### DISCUSSION TITLE
Configure the system locale to please `nvim +checkhealth` within `tmux`: Locale does not support UTF-8

### DISCUSSION THREAD

**Discussion by martin-braun** [termux/termux-app#3187] [Q&A]
I'm running `nvim` within `tmux` in a termux session. So far everything works, but the system locale check of `:checkhealth` complains: 

![ERROR: Locale does not support UTF-8. Unicode characters may not display correctly.
$LANG-en_US.UTF-8 $LC _ALL= $LC_ CTYPE=
ADVICE:
If using tmux, try the -u option.
Ensure that vour terminal/shell/tmux/etc inherits
the environment, or set $LANG explicitly.
Configure our system locale.](https://user-images.githubusercontent.com/24259245/212534961-fe4e70c1-33ad-4d23-88bd-2e7db67cf9c8.png)

I already launch tmux with the -u flag, $LANG is set explicitly. I fail to know how to "configure" the system locale in termux on Android. I couldn't find any information, there is no `dpkg-reconfigure locale` like in Debian. I assume I need to find the equivalent of termux, I do not? I did google the error, but often the issue was that $LANG was set to C or any other ASCII locale. Mine is properly set to an UTF 8 one.

What can I do to pass the health check?

**Comment by MarcoBuess** [comment]
Did you ever figure this out? I'm having the same issue.

**Reply by martin-braun** [reply]
Unfortunately, no, as I moved on. It was an experiment. I'm often with my laptop, but I felt it might be cool to get my `nvim` configuration to work on my phone. The idea was very pleasing to me, but Termux doesn't even play nice with `nvm`, too, so it made no sense to invest further. For simply checking files, I would not require to pass the health check.

Maybe the issue can be circumvented when running nvim within chroot in a sub system that is free of the Termux environment. At least this would be a solution for other stuff like `nvm`. I didn't took that route, though. I will probably revisit this idea when I acquire a foldable, which would really trigger me to want this working, as it would be a larger screen on the go.
